dev.yml

```diff
@@ -0,0 +1,19 @@
+# Meant to be used with docker-compose.yml
```

Comment:
Can we put it in some folder and name it more obviously, please? For example `rust/docker-compose.overrides-to-build-artifacts-locally.yml` or something shorter but without losing the meaning 🤣.

Our project root has a lot of stuff already and this is something not desirable for somebody developing a portal (no rust tools being built on the host)..